src/asyncActions/addTodo.js

```diff
@@ -5,15 +5,15 @@ import {
   setCompletedAllAction,
```

Comment:
Lets move asyncActions to one file "asyncActions.js" - asyncActions still not moved to one file